lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -553,7 +553,8 @@ unsigned TypeChecker::getCallEditDistance(DeclNameRef writtenName,
 
   // Don't typo-correct to a name with a leading underscore unless the typed
   // name also begins with an underscore.
-  if (correctedBase.starts_with("_") && !writtenBase.starts_with("_")) {
+  if (correctedName.getBaseIdentifier().hasUnderscoredNaming() &&
+      !writtenName.getBaseIdentifier().hasUnderscoredNaming()) {
```

Comment:
Since `correctedBase` and `writtenBase` are no longer used for this check, let's push their declarations down under it.

Reply:
Thank you. I’ve made the changes.
If everything looks good, could you run the CI bot ?